.npmrc

```diff
@@ -0,0 +1 @@
+auto-install-peers=true
```

Comment:
IMO we should install peer deps where necessary instead of avoid it with the auto-install like that, wdyt ?

Reply:
This has been added to solve an issue with a Docusaurus peer dependency. This allow to automatically install peer dependencies (which does not seem to be the default npm behaviour bty, cf. [here](https://pnpm.io/next/npmrc#peer-dependency-settings))

I'll add a comment to the file for context 📝 

![Screenshot 2022-12-22 at 17 44 58](https://user-images.githubusercontent.com/9203826/209186739-cf4074bd-e638-4beb-9734-f504857a8125.png)

Reply:
Can we have this file inside the `website` folder ?

Reply:
Yes but it won't solve the issue if we do `pnpm build` at the root of the monorepo